src/main/view/screen/IntroPanel.java

```diff
@@ -61,7 +61,7 @@ public void draw(GraphicsContext g, Point screenDimensions, long currentPulse) {
 			playIntroVideo();
 			introStarted = true;
 		} else {
-			if (videoPlayer.getCurrentTime().toMillis() == 39894) { //Last Milli of Video
+			if (videoPlayer.getCurrentTime().toMillis() == 39574) { //Last Milli of Video
```

Comment:
The new video is 320 milliseconds shorter. There was a glitch in the beginning where one of the pandas doesn't show up until 100 milliseconds in or so, so I fixed that too.